module-services/service-evtmgr/battery/BatteryController.cpp

```diff
@@ -122,18 +122,18 @@ void sevm::battery::BatteryController::poll()
 }
 void sevm::battery::BatteryController::printCurrentState()
 {
-    LOG_INFO("Charger state:%s Battery SOC %d voltage: %" PRIu32 "mV state: %s",
+    LOG_INFO("Charger state:%s Battery SOC %d voltage: %dmV state: %s",
              magic_enum::enum_name(Store::Battery::get().state).data(),
              Store::Battery::get().level,
-             charger->getBatteryVoltage(),
+             Store::Battery::get().voltage,
```

Comment:
I would leave it as it was before.

Reply:
Why? 
I moved the `charger->getBatteryVoltage()` to the `update()` function to save the voltage to `EventStore`. 
This approach was necessary because `CW2015` doesn't have a low voltage detection feature (brownout) 
so I had to save the voltage and then check if the value is critically low.
I wouldn't use `charger->getBatteryVoltage()` two times (for logging and checking low voltage) because it involves I2C to get the data.
If you see any reasons for changing this let me know :)

Reply:
`Store::Battery::get()` is simple a singleton in disguise. I'd suggest limit its usage to bare minimum. It would be nice to remove it altogether, but I know it's not possible for now.